package.json

```diff
@@ -19,7 +19,8 @@
   "dependencies": {
     "accepts": "^1.3.3",
     "debug": "^2.2.0",
-    "ejs": "^2.4.2",
+    "ejs": "^2.5.5",
+    "escape-html": "^1.0.3",
```

Comment:
This won't be needed after the changes in JSON output are reverted, please remove.